language/de/common.php

```diff
@@ -344,6 +345,9 @@
 	'MODERATE'				=> 'Moderieren',
 	'MODERATOR'				=> 'Moderator',
 	'MODERATORS'			=> 'Moderatoren',
+	'MODULE_NOT_ACCESS'		=> 'Modul nicht erreichbar',
```

Comment:
Passt das hier? "Modul not accessable" Ich würde eher in Richtung "Kein Zugriff auf das Modul möglich" gehen.